WinstonTransport types don't work anymore after latest release

After upgrading `fluent-logger` to version 3.4.0, the following types do not work anymore. Any suggestion on how to fix that?

Code:
```ts
import * as fluent from 'fluent-logger';
const FluentTransport = fluent.support.winstonTransport();
```

Error:
```ts
Property 'support' does not exist on type 'typeof fluentLogger'.
TS2694: Namespace 'fluentLogger' has no exported member 'FluentTransport'.
error TS2694: Namespace 'fluentLogger' has no exported member 'FluentTransport'.
```

--- a/lib/index.d.ts
+++ b/lib/index.d.ts
@@ -76,9 +76,9 @@ declare namespace fluentLogger {
     static fromTimestamp(t: number): InnerEventTime;
   }
 
-  interface Constructable<T, U> {
-    new(tag: string, options: U) : T;
-  }
+  let support: {
+    winstonTransport: any
+  };
 
   let EventTime: InnerEventTime;
 

